gatsby/content/projects/bots/hemppa.mdx

```diff
@@ -0,0 +1,18 @@
+---
+layout: project
+title: Hemppa
+categories:
+  - bot
+description: A multi-purpose Matrix bot
+repo: https://github.com/vranki/hemppa/
+featured: true
+language: Python
+author: cos
+license: GPLv3
+maturity: Production
```

Comment:
Production isn't a valid option so it won't show up. It has to be one of these words
<img width="234" alt="Screen Shot 2020-01-05 at 5 16 01 PM" src="https://user-images.githubusercontent.com/5855073/71787552-0d906200-2fdf-11ea-9353-f256fc8a13ad.png">